.github/workflows/web.yml

```diff
@@ -71,8 +73,79 @@ jobs:
               - '**/tests/clapi_export/**'
               - '**/www/!(front_src)/**'
               - '**/doc/API/**'
+            has_features_changes:
+              - '**/tests/e2e/**/*.feature'
           token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
 
+  gherkin-lint:
+    runs-on: ubuntu-22.04
+    needs: [changes]
+    if: ${{ needs.changes.outputs.has_features_changes == 'true' }}
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+
+      - name: Install Gherkin Lint
+        run: npm install gherkin-lint@4.2.2
+
+      - name: Add Gherkin Lint Rules
+        run: |
+          echo '
```

Comment:
can we commit this as a file instead?

Reply:
@kduret suggested the opposite to make the action as portable as possible.

Reply:
Yes, to be consistent with yaml linter : https://github.com/centreon/centreon/blob/develop/.github/workflows/actionlint.yml#L52-L66
But indeed, it could be commited as file, but could you move gherkin lint in a dedicated action ?

Then, you should have a dedicated directory : 
* `.github/actions/gherkin-lint/action.yml`
* `.github/actions/gherkin-lint/.gherkin-lintrc`

Sorry @a-eljazouly for asking you to change it again